src/commons/components/catalog-categories-item/catalog-categories-item.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+// style
+import './catalog-categories-item.css';
+
+const CatalogCategoriesItem = (props) => {
+  return(
+    <li className='catalog-categories-item'>
+      <button className={props.isActiveCategory === props.category ? 'catalog-categories-item-active' : null} onClick={(e) => props.categoryClick(e.target.innerText)}>{props.category}</button>
```

Comment:
if you insert a value from the properties inside the HTMLelement, then it is logical to pass it to the click event.
after dastructuring it will be to look.
onClick={() => categoryClick(category)}